quadrat/block-templates/search.html

```diff
@@ -3,10 +3,6 @@
 <!-- wp:group {"layout":{"inherit":true}, "tagName":"main"} -->
 <main class="wp-block-group">
 
-	<!-- wp:heading -->
-	<h2>Results:</h2>
```

Comment:
Should we do the same for blockbase?  That would make all 4 blockbase themes i11n-ready.